CRM/Contact/BAO/Contact.php

```diff
@@ -215,8 +215,8 @@ public static function add(&$params) {
       }
     }
 
-    // Update cached employer name.
-    if ($contact->contact_type === 'Organization') {
+    // Update cached employer name if the name of an existing organization is being updated.
+    if ($contact->contact_type === 'Organization' && !empty($params['organization_name']) && !$contactID) {
```

Comment:
Should `!$contactID` be `!empty($contactID)`, i.e. only do this when updating an existing org?

Reply:
opps yes - fixed